javascript/script.js

```diff
@@ -156,13 +176,15 @@ window.addEventListener("load", () => {
 	// }
 	let listReceived = JSON.parse(localStorage.getItem("listSaved")) ?? [];
 	if (listReceived.length === 0) {
-		let emptyImage = "<figure id='empty-image'><img src='./images/clipboard.svg' class='empty'><figcaption>Nothing here. Add tasks to view here.</figcaption></figure>";
-		$("list-items").insertAdjacentHTML('afterbegin', emptyImage);
+		let emptyImage = $('empty-image-template').content.cloneNode(true)
+		$("list-items").append(emptyImage);
```

Comment:
I see that you're using `$("list-items")` considerably a large number of times. Extract it into a global variable and re-use it everywhere.